readme.md

```diff
@@ -47,126 +62,497 @@ This package exports [ESM](https://gist.github.com/sindresorhus/a39789f98801d908
 
 ## Usage
 
-This is a basic example using OpenAI's [text-embedding-ada-002](https://platform.openai.com/docs/guides/embeddings) embedding model and a [Pinecone](https://www.pinecone.io/) datastore to index and query a set of documents.
+Here's a basic example of how to use the ChatModel:
 
-```ts
-import 'dotenv/config';
-import { EmbeddingModel } from '@dexaai/dexter/model';
-import { PineconeDatastore } from '@dexaai/dexter/datastore/pinecone';
+```typescript
+import { ChatModel } from '@dexaai/dexter';
 
-async function example() {
-  const embeddingModel = new EmbeddingModel({
-    params: { model: 'text-embedding-ada-002' },
-  });
+const chatModel = new ChatModel({
+  params: { model: 'gpt-3.5-turbo' },
+});
 
-  const store = new PineconeDatastore({
-    contentKey: 'content',
-    embeddingModel,
+async function main() {
```

Comment:
no need for a `main` function in the main example; use top-level wait; brevity is key especially for the first example a dev sees